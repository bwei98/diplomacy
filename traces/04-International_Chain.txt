# Spring 1901
F Lon - ENG






# Fall 1901
F ENG - Bre
F Bre - MAO





#
# END