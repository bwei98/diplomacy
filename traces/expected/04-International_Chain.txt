# Spring 1901
F ENG; A Lvp; F Edi
pass
# Fall 1901
F Bre; A Lvp; F Edi
Lvp, Edi, Lon
F MAO; A Par; A Mar
pass
#
# END